app/controllers/users_controller.rb

```diff
@@ -1,3 +1,27 @@
 class UsersController < ApplicationController
-  def new; end
+  def show
+    @user = User.find params[:id]
+  end
+
+  def new
+    @user = User.new
+  end
+
+  def create
+    @user = User.new user_params
+
+    if @user.save
+      flash[:danger] = t "static_page.user.welcome"
+      redirect_to @user
+    else
+      render :new
+    end
+  end
+
+  private
+  
+  def user_params
+  	params.require(:user).permit(User::PERMIT)
```

Comment:
```suggestion
  	params.require(:user).permit User::PERMIT
```
e nên đặt lại const là USER_PARAMS nhé